apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.env.$envParam.queues/route.tsx

```
@@ -0,0 +1,615 @@
+
+  const formData = await request.formData();
+  const action = formData.get("action");
+
+  const url = new URL(request.url);
+  const { page } = SearchParamsSchema.parse(Object.fromEntries(url.searchParams));
+
+  const redirectPath = `/orgs/${organizationSlug}/projects/${projectParam}/env/${envParam}/queues?page=${page}`;
+
+  switch (action) {
+    case "environment-pause":
+      const pauseService = new PauseEnvironmentService();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)